catalog/app/containers/Bucket/File.js

```diff
@@ -306,28 +315,24 @@ const useStyles = M.makeStyles((t) => ({
     display: 'flex',
     marginBottom: t.spacing(2),
   },
-  at: {
-    color: t.palette.text.secondary,
-  },
-  spacer: {
-    flexGrow: 1,
-  },
-  button: {
-    flexShrink: 0,
-    marginBottom: -3,
-    marginTop: -3,
-  },
 }))
 
+const viewModes = [
+  { key: 'jupyter', toString: () => 'Jupyter', valueOf: () => 'jupyter' },
+  { key: 'voila', toString: () => 'Voila', valueOf: () => 'voila' },
+  { key: 'json', toString: () => 'Json', valueOf: () => 'json' },
```

Comment:
do we really need these `toString` and `valueOf` functions? seem like smth like `{ key: 'jupyter', label: 'Jupyter' }` would be enough